packages/vector_graphics_compiler/lib/src/svg/node.dart

```diff
@@ -124,10 +124,40 @@ class ParentNode extends Node {
     );
   }
 
+  // Whether or not a save layer should be inserted at this node.
+  bool _requiresSaveLayer() {
```

Comment:
This would be much simpler if I knew the original properties on the group instead of having to reverse-engineer them from the paint